recipes/gnss-sdr/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set version = "0.0.18" %}
+
+package:
+  name: gnss-sdr-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/gnss-sdr/gnss-sdr/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d5533370de188d50c214a5bb7832a433ceb09369c3e3296a4fa34fa1aefb685e
+  patches:
+    - 0001-Manually-specify-compiled-in-GNSSSDR_INSTALL_DIR.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
+  - name: volk-gnss-sdr
+    script: build-volk-gnss-sdr.sh  # [unix]
+    script: bld-volk-gnss-sdr.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - cmake
+        - gstreamer-orc  # [not win]
+        - mako
+        - ninja
+        - pkg-config
+        - python >=3.7
+      host:
+        - gstreamer-orc  # [not win]
+      run:
+        - {{ pin_compatible('gstreamer-orc', max_pin='x.x') }}  # [not win]
+    test:
+      commands:
+        # verify that commands run
+        - volk_gnsssdr-config-info --version --prefix --cc --cflags --all-machines --avail-machines --machine --alignment --malloc
+        - volk_gnsssdr_profile --help
+
+        # verify libraries
```

Comment:
Please also verify existence of some headers / cmake files / config files.

Reply:
Done!